### DISCUSSION TITLE
Upgrading from 1.4.1 to 2.0.0

### DISCUSSION THREAD

**Discussion by Saxon613** [flyntwp/flynt#498] [Q&A] [answered]
Hi there, the new version looks awesome and I'm looking forward to building our next site on it.

Down the line, I may look into upgrading some of our 1.4.1 sites. Do you plan on providing any documentation/support on the best approach to do this? How painful do you think it will be?

Thanks!
S

**Comment by steffenbew** [comment, marked as answer]
Hey @Saxon613, I'm glad you like the the release!

Flynt is meant to be a boilerplate for projects and that's why we typically do not upgrade bespoke themes that are created from it.

There are exceptions to the rule, where we like to migrate some specific features or fixes. Like you might be able to use the new component search functionality from [FeatureFlexibleContentExtension](https://github.com/flyntwp/flynt/tree/master/Components/FeatureFlexibleContentExtension).

But with v2.0 being an extensive upgrade that changes a couple of core concepts and technologies, I won't consider upgrading any of our v1.4.1 to v2.0 themes entirely.

**Reply by Saxon613** [reply]
Understood! Thanks a lot for the response, and of course the good work.

Cheers,
S
